tuplex/core/src/DataSet.cc

```diff
@@ -38,21 +38,21 @@ namespace tuplex {
     }
 
     std::shared_ptr<ResultSet> DataSet::collect(std::ostream &os) {
-        return take(-1, os);
+        return take(-1, false, os);
     }
 
-    std::shared_ptr<ResultSet> DataSet::take(int64_t numElements, std::ostream &os) {
+    std::shared_ptr<ResultSet> DataSet::take(int64_t numTop, int64_t numBottom, std::ostream &os) {
```

Comment:
use size_t here instead of int64_t? watch out for -1 case! Is -1 somewhat used for take throughout the code-base? Thorougly check before making the change. I.e., especially check in PythonDataSet.cc....